label_studio/ml_models/migrations/0011_thirdpartymodelversion_model_provider_connection.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 3.2.25 on 2024-09-12 21:59
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
```

Comment:
To make it backward compatible, we need to add migrations that would relate all previously existing model connections to the model versions. Then we can safely get rid of the current logic in the code

Reply:
Check this as an example https://github.com/HumanSignal/label-studio/blob/develop/label_studio/data_manager/migrations/0010_auto_20230718_1423.py#L42